.travis.yml

```diff
@@ -1,10 +1,28 @@
 language: node_js
 node_js:
   - "8"
-  - "9"
```

Comment:
We should probably test it on version 10 also

Reply:
But I guess this doesn't really matter as it's testing inside a docker container...